File: src/main/java/com/p2p/lending/dao/DetailsDao.java
package com.p2p.lending.dao;

import java.util.List;

import com.p2p.lending.entity.Details;

public interface DetailsDao extends  BaseDao<Object, Details>  {
	public List<Details> detailslist(Integer ids);

	public void detailsadd(Details notice);
	
	public Details detailsget(Integer ids);
	
	public void detailsdel(Integer ids);
	public void deleteByPid(Integer pid);
	
	/*


	public void detailsupd(Details notice);

	*/

}

File: src/main/java/com/p2p/lending/dao/DetailsDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Details

import org.springframework.stereotype.Repository

@Repository
interface DetailsDao : BaseDao<Any, Details> {
    fun detailslist(ids: Int?): List<Details>

    fun detailsadd(notice: Details)

    fun detailsget(ids: Int?): Details

    fun detailsdel(ids: Int?)

    fun deleteByPid(pid: Int?)


}
